lang_python.adoc

```diff
@@ -13,98 +13,140 @@ To generate Python modules, Kaitai Struct compiler should be invoked with
 `-t python` option:
 
 [source,shell]
-kaitai-struct-compiler -t python my_spec.ksy
+kaitai-struct-compiler -t python --python-package . my_spec.ksy
 
 This normally generates `my_spec.py` parser module, which is compatible
-with both Python 2 and 3.
+with both Python 2 and 3. It is strongly recommended to use python 3
+because python 2 is EOL since 2020-01-01. You can also install
```

Comment:
Python should be written with the first uppercase letter (it's a name), writing it somehow else in a normal text looks unprofessionally.

```suggestion
with both Python 2 and 3. It is strongly recommended to use Python 3
because Python 2 is EOL since 2020-01-01. You can also install
```

Reply:
Thanks, fixed.